direct/io-bulk-fs/src/main/java/cz/o2/proxima/direct/bulk/BinaryBlobFormat.java

```diff
@@ -116,22 +116,29 @@ public void close() throws IOException {
         blobStream = null;
       }
     }
+
+    @Override
+    public Path getPath() {
+      return path;
+    }
   }
 
-  public static class Reader implements Iterable<StreamElement>, Closeable {
+  public static class BinaryBlobReader implements Reader {
```

Comment:
nit: I'd prefer `  public static class Reader implements cz.o2.proxima.direct.bulk.Reader {` (same applies to writer)

Reply:
I don't like classes extending other classes with same name, although in different package. I'll keep as is.